_data/sites.json

```
@@ -3091,6 +3091,16 @@
         ]
     },
 
+    {
+        "name": "Brandpad",
+        "url": "https://create.brandpad.io/account/delete-account",
+        "difficulty": "easy",
+        "notes": "Account-Circle on the top right > Account > Delete Account --- You need to cancel your subscription first --- Deleting your account does not automatically unpublish or delete the brands connected to your account.",
```

```suggestion
        "notes": "You need to cancel your subscription first, as deleting your account does not automatically unpublish or delete the brands connected to your account. Then on Account-Circle on the top right ⮕ Account ⮕ Delete Account.",
```
Moved the first part to actually be the first on the notes.